src/main/kotlin/uk/gov/dluhc/printapi/database/entity/AnonymousElectorDocument.kt

```diff
@@ -41,40 +41,40 @@ class AnonymousElectorDocument(
 
     @field:NotNull
     @field:Size(max = 20)
-    var certificateNumber: String? = null,
+    var certificateNumber: String,
 
     @field:NotNull
     @Enumerated(EnumType.STRING)
-    var sourceType: SourceType? = null,
+    var sourceType: SourceType,
 
     @field:NotNull
     @field:Size(max = 255)
-    var sourceReference: String? = null,
+    var sourceReference: String,
 
     @field:Size(max = 255)
-    var applicationReference: String? = null,
+    var applicationReference: String?,
 
     @field:NotNull
     @Enumerated(EnumType.STRING)
-    var certificateLanguage: CertificateLanguage? = null,
+    var certificateLanguage: CertificateLanguage?,
```

Comment:
Minor. I think we can remove nullable for this field as well.
<pre>
@field:NotNull
@Enumerated(EnumType.STRING)
var certificateLanguage: CertificateLanguage,
</pre>

Reply:
Yeah - I think you are right, have updated the schema too.